pydata/views.py

```diff
@@ -0,0 +1,124 @@
+import requests
+
+from django.views.generic import View
+from django.http import JsonResponse, HttpResponseBadRequest
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.core.exceptions import ObjectDoesNotExist
+
+from workshops.models import Event, Tag, Role
+from workshops.views import TaskCreate as BaseTaskCreate
+from workshops.views import PersonCreate as BasePersonCreate
+
+from .forms import ImportTaskFullForm, ImportPersonForm
+from .models import Person, Task
+
+
+class BaseImport(LoginRequiredMixin, View):
```

Comment:
I think you need to use some other mixin because we changed the logging permissions.